test/Libraries/PackageManagerTests/PackageTests.cs

```diff
@@ -64,7 +64,7 @@ public void IsNodeLibrary_IsFalseAndHasMessagesForBadlyFormatted()
             IList<ILogMessage> ws = new List<ILogMessage>();
             Assert.False(Package.IsNodeLibrary(new List<string>
             {
-                "\\ x x x x x x"
+                "\\ x \\ x x x x x"
```

Comment:
1. I don't understand this change - can you explain it?
2. you have not removed the failure category so the test is not being run in the PR checks.

Reply:
@mjkkirschner, answering your questions:

1. When I was running the unit test on Dynamo.All.Net6.sln, it wasn't passing because that string was valid, I had to try different strings until I found that.
2. Sorry, I didn't know that, I will fix it.

Reply:
Done.